src/wireviz/Harness.py

```diff
@@ -344,18 +349,15 @@ def bom(self):
         # bundles (ignores wirecount)
         wirelist = []
         # list all cables again, since bundles are represented as wires internally, with the category='bundle' set
-        bundle_group = lambda b: (b.category, b.type, b.gauge, b.gauge_unit, b.length)
-        groups = Counter([bundle_group(v) for v in self.cables.values() if v.category == 'bundle'])
-        for group in groups:
-            items = {k: v for k, v in self.cables.items() if bundle_group(v) == group}
-            shared = next(iter(items.values()))
-            for bundle in items.values():
+        # TODO: Maybe rename bundle to cable or maybe loop only [c for c in self.cables.values() if c.category == 'bundle']
```

Comment:
I think bundle is fine, since that is what we are looking for. If becomes clear after `if bundle.category == 'bundle':` at the latest.